spec/async/http/cache/general_spec.rb

```diff
@@ -68,43 +68,113 @@
 				else
 					Protocol::HTTP::Response[200, [['cache-control', 'max-age=1, public'], ['vary', 'user-agent']], ['Hello', ' ', 'World']]
 				end
-				
+
 				if request.head?
 					response.body = Protocol::HTTP::Body::Head.for(response.body)
 				end
-				
+
 				response
 			end
 		end
-		
+
 		let(:user_agents) {[
 			'test-a',
 			'test-b',
 		]}
-		
+
 		it "should cache GET requests" do
 			2.times do
 				user_agents.each do |user_agent|
 					response = subject.get("/", {'user-agent' => user_agent})
 					expect(response.headers['vary']).to include('user-agent')
-					expect(response.read).to be == user_agent
+					expect(response.read).to eq(user_agent)
```

Comment:
I've never seen `be ==`. Is that preferred in this project vs simply `eq()`?

Reply:
Yeah, I don't like abbreviations and don't find `eq` very intuitive.